synapse/eventbus.py

```diff
@@ -25,6 +54,26 @@ def __init__(self):
 
         self._fini_funcs = []
 
+        for name,valu in s_reflect.getItemLocals(self):
+
+            if not callable(valu):
+                continue
+
+            # check for onfini() decorator
+            if getattr(valu,'_ebus_onfini',None):
+                self.onfini(valu)
+                continue
+
+            # check for on() decorators
+            eons = getattr(valu,'_ebus_ons',None)
```

Comment:
agreed with a small tweak, use of [] ( list ) as the default value causes the construction of a new empty list object every time the line is run.  However, with a tuple () ( since it's immutable ) the interpreter only constructs it once. will tweak.